content/en/serverless/google_cloud_run/_index.md

```diff
@@ -10,183 +10,508 @@ further_reading:
+WORKDIR /app
+COPY ./bin/Release/net8.0/publish /app
 
-The following command deploys the service and allows any external connection to reach it. Set `DD_API_KEY` as an environment variable, and set your service listening to port 8080.
+ADD https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.56.0/datadog-dotnet-apm_2.56.0_amd64.deb /opt/datadog/datadog-dotnet-apm_2.56.0_amd64.deb
+RUN dpkg -i /opt/datadog/datadog-dotnet-apm_2.56.0_amd64.deb
+RUN mkdir -p /shared-volume/logs/
 
+ENV CORECLR_ENABLE_PROFILING=1
+ENV CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+ENV CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
+ENV DD_DOTNET_TRACER_HOME=/opt/datadog/
+
+ENV DD_TRACE_DEBUG=true
+
+ENTRYPOINT ["dotnet", "dotnet.dll"]
 ```
-shell
-gcloud run deploy APP_NAME --image=gcr.io/YOUR_PROJECT/APP_NAME \
-  --port=8080 \
-  --update-env-vars=DD_API_KEY=$DD_API_KEY \
-  --update-env-vars=DD_TRACE_ENABLED=true \
-  --update-env-vars=DD_SITE='datadoghq.com' \
-  --update-env-vars=DD_TRACE_PROPAGATION_STYLE='datadog' \
+
+#### Metrics
+The tracing library also collects custom metrics. See the [code examples][2].
+
+#### Logs
+The Datadog sidecar collects logs through a shared volume. to forward logs from your main container to the sidecar, configure your application to write all logs to a location such as `shared-volume/logs/*.log`. 
+
+To set up logging in your application, see [C# Log Collection][3]. To set up trace log correlation, see [Correlating .NET Logs and Traces][4].
+
+[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-core/?tab=linux#enable-the-tracer-for-your-service
+[2]: /metrics/custom_metrics/dogstatsd_metrics_submission/#code-examples
+[3]: /log_collection/csharp/?tab=serilog
+[4]: /tracing/other_telemetry/connect_logs_and_traces/dotnet/?tab=serilog
+{{% /tab %}}
+{{% tab "PHP" %}}
+In your main application, add the `dd-trace-php` library. See [Tracing PHP applications][1] for instructions.
+
+#### Metrics
+The tracing library also collects custom metrics. See the [code examples][2].
+
+#### Logs
+The Datadog sidecar collects logs through a shared volume. to forward logs from your main container to the sidecar, configure your application to write all logs to a location such as `shared-volume/logs/*.log`. 
+
+To set up logging in your application, see [PHP Log Collection][3]. To set up trace log correlation, see [Correlating PHP Logs and Traces][4].
+
+[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/php/
+[2]: /metrics/custom_metrics/dogstatsd_metrics_submission/#code-examples
+[3]: /logs/log_collection/php
+[4]: /tracing/other_telemetry/connect_logs_and_traces/php
+{{% /tab %}}
+{{< /tabs >}}
+
+### Sidecar container
+
+1. In Cloud Run, select **Edit & Deploy New Revision**.
+1. At the bottom of the page, select **Add Container**.
+1. For **Container image URL**, select `gcr.io/datadoghq/serverless-init:1.2.5`
```

Comment:
Can we potentially update this to be a link to the latest stable version of serverless-init instead of a hardcoded `1.2.5` version